composer.json

```diff
@@ -193,6 +193,7 @@
         "spryker/stores-api": "^1.0.0",
         "spryker/stores-backend-api": "^1.0.0",
         "spryker/stores-rest-api": "^1.2.0",
+        "spryker/tax-app": "^0.1.0",
```

Comment:
I couldn't find this 2 packages listed on the PBC requirements (do we assume are included in some of the spryker-feature packages? If that is the case, how do we know the package included on the feature maps with the required version?):
- spryker/stock-address: ^1.2.0
- spryker-shop/cart-page: ^3.38.0

Reply:
The two packages are already required by other packages with compatible constraints.
```
mads@xps:~/projects/demoshops/b2c-demo-shop$ composer why spryker/stock-address
spryker-feature/inventory-management 202307.x-dev requires spryker/stock-address (^1.1.0) 
spryker/stock-address-data-import    0.1.1        requires spryker/stock-address (^1.0.0) 
mads@xps:~/projects/demoshops/b2c-demo-shop$ composer why spryker-shop/cart-page
spryker-feature/cart 202307.0 requires spryker-shop/cart-page (^3.36.0) 
```
So the package update is only contained as part of the composer.lock file.